sig/engine/constitution-zh_CN.md

```diff
@@ -0,0 +1,80 @@
+# Engine SIG 章程
+
+本章程遵循 TiKV Community 章程和制度，使用 [SIG Governace](/GOVERNACE-zh_CN.md) 中的角色定义以及职责。
+
+## 职责范围
+
+Engine SIG 的主要职责是对 TiKV 的存储引擎的未来发展进行讨论和规划，并组织社区成员进行相关开发和维护。工作范围包括为 TiKV 引入新的存储引擎，以及对现有存储引擎进行稳定性、性能、易用性等方面的提升。仅 Active Contributor 或更高等级社区成员可成为 SIG 成员并参与到相应事项中来，但 SIG 各项实行的决策和进行的项目将公开。
+
+## 工作内容
+
+- 代码范围
+  - Engine Traits: 存储引擎抽象 [engine_traits](https://github.com/tikv/tikv/tree/master/components/engine_traits)。
+  - Engine Rocks: RocksDB 存储引擎封装 [engine_rocks](https://github.com/tikv/tikv/tree/master/components/engine_rocks)。
+  - Engine: 旧版 RocksDB 存储引擎封装 [engine](https://github.com/tikv/tikv/tree/master/components/engine)。
+  - Rust RocksDB: Rust RocksDB 接口封装 [rust-rocksdb](https://github.com/tikv/rust-rocksdb)。
```

Comment:
btw shall we also move pingcap/rocksdb to under tikv org? @siddontang

Reply:
I think it is ok to move to tikv org.

Reply:
I think we need a discussion about it.

Reply:
Maybe we can add `pingcap/rocksdb` and `pingcap/titan` here. Engine SIG is working on the project in PingCAP is also reasonable.

Reply:
If engine traits is ready now, it would be very useful to add more docs or a demo example about adding a new engine other than rocksdb. Then it is easier for contributors to try new ideas about new engine.

Reply:
@fredchenbj AFAIK engine trait is not ready but contribution to code and docs are welcome.

Reply:
> Maybe we can add `pingcap/rocksdb` and `pingcap/titan` here. Engine SIG works for the projects in PingCAP is also reasonable.

I suppose we will see more and more store engines which are maintained separately being added to TiKV in the future. Therefore it is unnecessary to move `pingcap/rocksdb` and `pingcap/titan` here.